conduce.py

```diff
@@ -42,6 +42,17 @@ def find_resource(args):
     return resources
 
 
+def open_in_editor(content):
+    EDITOR = os.environ.get('EDITOR', 'vim')
+    with tempfile.NamedTemporaryFile(suffix='.tmp') as resource_file:
+        resource_file.write(str(content))
+        resource_file.flush()
+        call([EDITOR, resource_file.name])
+
+        with open(resource_file.name, 'r') as edited_resource_file:
```

Comment:
Git has this thing with interactive rebases:

`# if you remove everything, the rebase will be aborted.`

It would be great to do something like that here, where there is a documented way to do nothing

Reply:
I thought the same thing.  Right now, if the edited version is string equivalent to the original, nothing will happen.

I supposed we could also add a feature to exit gracefully if the user deletes the entire contents and attempts to commit those changes.

I'm not sure how we would document it though, outside of putting it in argparse.

Reply:
Also, deleting the contents of the file and saving it should have the desired effect, although I suspect the empty string will be sent to Conduce before failing.